bundles/org.openhab.binding.mqtt/src/main/java/org/openhab/binding/mqtt/handler/BrokerHandler.java

```diff
@@ -223,8 +223,6 @@ protected MqttBrokerConnection createBrokerConnection() throws IllegalArgumentEx
             connection.setTimeoutExecutor(scheduler, TIMEOUT_DEFAULT);
         }
 
-        connection.setRetain(config.retainMessages);
```

Comment:
Should the config also be removed or is it used elsewhere?

Reply:
It's still used in other locations of this file.

Reply:
oh ok